cmd/krel/cmd/release_notes.go

```diff
@@ -167,6 +168,14 @@ func init() {
 		"add dependency report",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringSliceVarP(
+		&releaseNotesOpts.mapProviders,
+		"maps-from",
+		"m",
+		[]string{},
+		"specify a location to recursively look for release notes maps",
```

Comment:
I suggest to hint for the file extension:
```suggestion
		"specify a location to recursively look for release notes *.y[a]ml file mappings",
```

Reply:
Yes much clearer. done.